cfme/configure/settings.py

```diff
@@ -62,26 +62,29 @@ def __init__(self, description=None, scope=None, days=None, hours=None, timezone
         self.hours = hours
         self.timezone = timezone
 
-    def create(self):
+    def create(self, negative=False):
```

Comment:
@psav @mshriver In this case we don't press "cancel" button, only filling the form. I add this for negative tests.  But okay, i will change this